packages/mobile/src/identity/contactMapping.ts

```diff
@@ -39,13 +43,37 @@ import { getConnectedAccount } from 'src/web3/saga'
 import { currentAccountSelector } from 'src/web3/selectors'
 
 const TAG = 'identity/contactMapping'
+export const IMPORT_CONTACTS_TIMEOUT = 2 * 60 * 1000 // 2 minutes
```

Comment:
What's the thought on this length? Seems a bit on the long side to me.

Reply:
Yeah its def a bit on the long side. The thought was to be cautious until we have better estimates of how long matchmaking will take. It'll hopefully be < 10 seconds.